Makefile

```diff
@@ -101,7 +101,7 @@ develop: tags
 	$(PIP) install -e .[doc,test]
 
 test:
-	$(COVERAGE) run -m $(PYTEST) tests -v
+	$(COVERAGE) run -m $(PYTEST) tests -v -r sx
```

Comment:
I considered it, but I'm quite liking the fact it now outputs precisely why it skipped particular tests - any objections to keeping it in?